unit_tests/src/test_ticket_administration.cpp

```diff
@@ -0,0 +1,365 @@
+
+        bool test_ticket_exists = false;
+
+        for (const auto& row : general_query) {
+            if (row[0] == read_ticket_name) {
+                test_ticket_exists = true;
+                break;
+            }
+        }
+
+        REQUIRE(test_ticket_exists);
+
+        std::string query =
+            fmt::format("select TICKET_COLL_NAME, TICKET_TYPE where TICKET_STRING = '{}'", read_ticket_name);
+        general_query = builder.build<rcComm_t>(conn, query);
+
+        REQUIRE(read_coll_string == (general_query.begin().capture_results().at(0)));
+        REQUIRE(read_string == (general_query.begin().capture_results().at(1)));
+    }
+
+    SECTION("delete ticket")
+    {
+        tic::client::delete_ticket(conn, read_ticket_name);
+
+        auto general_query = builder.build<rcComm_t>(conn, fmt::format("select TICKET_STRING where TICKET_STRING = '{}'", read_ticket_name));
+
+        bool test_ticket_exists = false;
+
+        for (const auto& row : general_query) {
+            if (row[0] == read_ticket_name) {
+                test_ticket_exists = true;
+                break;
+            }
+        }
+
+        REQUIRE(!test_ticket_exists);
+    }
+
+    SECTION("create write ticket")
+    {
+        tic::client::create_ticket(conn, tic::ticket_type::write, write_coll_string, write_ticket_name);
+
+        auto general_query = builder.build<rcComm_t>(conn, fmt::format("select TICKET_STRING where TICKET_STRING = '{}'", write_ticket_name));
+
+        bool test_ticket_exists = false;
+
+        for (const auto& row : general_query) {
+            if (row[0] == write_ticket_name) {
+                test_ticket_exists = true;
+                break;
+            }
+        }
+
+        REQUIRE(test_ticket_exists);
+
+        std::string query =
+            fmt::format("select TICKET_COLL_NAME, TICKET_TYPE where TICKET_STRING = '{}'", write_ticket_name);
+        general_query = builder.build<rcComm_t>(conn, query);
+
+        REQUIRE(write_coll_string == general_query.begin().capture_results().at(0));
+        REQUIRE(write_string == general_query.begin().capture_results().at(1));
+    }
+
+    SECTION("admin delete ticket")
+    {
+        tic::client::delete_ticket(tic::admin_tag{}, conn, write_ticket_name);
+
+        auto general_query = builder.build<rcComm_t>(conn, fmt::format("select TICKET_STRING where TICKET_STRING = '{}'", write_ticket_name));
+
+        bool test_ticket_exists = false;
+
+        for (const auto& row : general_query) {
+            if (row[0] == write_ticket_name) {
+                test_ticket_exists = true;
+                break;
+            }
+        }
+
+        REQUIRE(!test_ticket_exists);
+    }
+
+    SECTION("admin create read ticket")
+    {
+        tic::client::create_ticket(tic::admin_tag{}, conn, tic::ticket_type::read, read_coll_string, read_ticket_name);
```

Comment:
Instantiating the `admin_tag` is fine, but you introduced a global variable that does the same thing.

Prefer using `tic::admin` over `tic::admin_tag{}`.

This applies to all places where you use `tic::admin_tag{}`.